notebooks/data_viz_easy/raw/ex1.ipynb

```diff
@@ -423,87 +179,32 @@
UA7DyYslWjOeDRnF0HReWvl8QDxLFBe1IQYG4HONYuWvagw8DRcaxMe59VBp02cRwkJ08hdf166JKS4DBsGKr/ugjir7+GwNIKLhPGw6xePSgfP4FZTU+YeniUOuWg1O8C4r//HlZtWsNzwwbITDn8cnMe+kbcQvekSEAgglpkht+c7NBdQ2g76RHLQ3/qe3ZPikwBoSnLR9zhx0KPcTfxLhbdXoRNXTehrn1dPEh6gO3PtmNdp3UFlWsiYHt3ICsG+O5R3r1OBEReBpKeAcnP2ZIWxgZHP8ezgcveIcxtyMKBuQG9WrrMZ0aM6FvArfXsvhcIAa9eoGZjkHLiCTJ27oS5tzdqrFkD83p184kku34dCT/PBZ+dDfdf5sFx6P+G4pd54CBS//gDXEYGTGvWhOOwQXDGYeaCNi0YsK/B8vHbeRZ9X2iVwMaPARMLqLv9g/SduyE5fRrV5v4M+8GDoYmOzk3lSzwP+Y0byNizF/KbN+H55wZYtmgBbXwcLO/8wNLxTnkAWDtX3pfwP45Rqa4iSKtF6saNSP/rbziNGQ33n35iD7G0iAL+lEXBSaUQWlkVaZ2K7N0HpjVqoNbWLeUp+rtHeiSzxnSY+fYo+0WR+BTwHc7yRA/ZxiocGqk80l4A15czq51pCT6t5UHiU/BbOkHbcTbMO80psfkfj/6Atak1vmnyDQBAevEi5LduodrcuQZVvysSTgtsaJ6n0JrbAT9GMuVYIy9d4R2Azbxc+41ZVxOe5OVBt7AHZkUxC/bL60zpdqoPPNnHcu++sjzrS9g5ZkEvZV59XqWC4v59iJydYdGwIeK++z9oYqJRfdEigOcR82WOJVcohGm1avBYsZwp20+ewOKDD2Di7l6k/zLxPHQ5FsX0bdvhPHECeBMh4rLjUMe+DrM+2rgBHj5QaBUYfW40etfrja8bF1/MpTDCM8OxJmANVnRYAXtze1yLvYb1j9Zjc9fNqGZdyExI7ANge1eW896hVl7BrzUfANI4wK4GG1C5f8AW7z7s/uC0TMkvicxo4OEuZrVWpAPjrkL2IhsJs+fA6uOP4bl+XW7TjH37kLx4CcwbNIDHqpWwaPDa+1CnZoVOXN7L+d9Af+K3AF6jQfbFi8jy3QeRNBiereKQktEJVK0V7AcNhNmZz6FOlMK0WVeIWgyEoH7n3CqpAICry0DXliM+vi+ybz6CwMICDp99BuevxuZW0Hw1m6XQKmBlygwF6pdRMKtdC+lbtyJ13XrYfNISNeucAxoPZoXDygtFBhBxEciIZM+Mll8zI2HQYeDpfjYo0MiZoUurBD6ewGYqs8Rs5sPlfWZwcX7vnSxUY1SqqxhtcjJMTFUQZEWxICEDkZw4Acmp06i5dUu+h7k2KQkvOn0Kt1mz4Pz1V+Un8LvOy2uAi9fbabEOOcUCLK2cgREHgGqNq1qi/x4hp4CDXzKlYdju0rkr6ImW02LU2VEY6/EpevtM0HvqlyceQWlBaOpafJByqQn3Y0rt+z3YoK60hZ+KgueBjJdMwZYlsRcqEbCyPlO4TCyYy1DttsCAjYBT3ZL7fBNpIit00WNpub2YebkcqtBQaMSx0MaKoRHHwm3aD1A+fYr46TMgtLJCgwf3IT1zhlkLGzdmirabG6DTIXHePMhu3ES9kydy3RiO3VqKhS8OYH+fA2jkkr+qqppTw1xkDr9oP7Sr0S63KFBxiKVi/PH4DyxssxA2Zvlnl3S8DhmqDNxLvId+9QupdHh8MvDsCAsy//oCux6Tn7MZnPK6BnRqVgnYqxcgEEC7sQ8E9h7QefZC8o4TcJ83FyIbG2Tu3w+XyZMhDD+R42oYzj4zo5lhZG4SU+Z39gESn7DBiE019mlbDWg9CXCsA2TGsJkxK2e2mL4llRIVGcDu/qCkMPD9tyB6/nZo4+LhuXYtRNLniJ7NBjUCEx6m1oTqQxrD9Iv1yNq/G3RrK9xGdkZyRAMIbWzg+MWofK5KviG+uBZ7DQ2dG+Ja7DXs7b0XtmZ5lUg5qRSSY8cAAI414xC7/ABEH3QGIALxPOx69YJt1y6ImzIVxPNwn/UjeLUGKb//DqG1Nex694Jt164QWr5hdAg6zAZOMbfZgBwCwNQKGHmAzcA93MVKqZtZswGZqRVbvPsADfsCYeeB/cORWyFaIGTJEBoPZjMfPAckP2ODuwp8LpcVfZVqY0VFQ7n6G2DjzkZoTvXZSL+Ql6apmxuwZyAQFwD8EGTww4t4gvzmTSjuP4D1x3kR7OqIFxCYm8P6kzZlPpX/GVRS4NBY9vD96mz5Kwtl4ekB4NgEoEZLYLhv+VRHNGI4DfsBvVawLCe7+rL719WLKX4A8PQgezGb27FME+b27FOPLAz5UGYi69E/cLF0gV21JgZlYNjxbAc2PN6AkwNPorZdMe4QryGWijHy7Ej82PJHDHhvQPGNG/RgS0UhFDJL4ytr4yvGnMqzZLs3ApqPLX1miugbwIOtLCZg6D/lYs0UWlvDqkULWLVokW+9yNkZdQ7shzYpGQKRCIqHj5B16BAbPACwaNIENdaugey6P5zGjIbIOWeqPegwOlxdj+nutdDQumAmIHOROaIl0ZjtPxuTmk7KLVVfHOGZ4XiU/AjpqvQCSrWJ0ATbgrbhZORJtK3RFk4Wbzz/Bm5ig5jXr2P3/Ip+mTExzyujzWlhWut9IPBfSM+dhuKhA1Knj0LNkfXgNuMgG2idnwOos5nV0v0DpmC5egPEvls0/Ryo/iGQnQTIkpnS9eIy4PMF237vL+Duprzjm1ozV4eOcwCfUcy15ck+9i6wrwnUalO8i1F5ceQbIDUcglH7cc1cgOPT6+Lnj7bC2sodvLQZavzxPnTxcdCGBkAb+RxCkQqa6Fikbd0LUxsLOLf5Ce6fFcwMBgDWptawMbNB6+qtoeE0sDLJ79ImsrOD05gxAAAuPRWo+RLK56EQqLMAjQycZTTgKIYuQQyYW4O0WgAA8RzUYWFIuHoVQhsb1P1rKUwyH0
```

Comment:
These really long lines are github's representation of the images. They should get removed when you run `python clean.py data_viz_easy`.

Since I made the PR on github, I could have done it on your branch before making the PR. Anyway, the clean.py function is nice for making the diffs shorter and more readable.